load-fhir-data.js

```diff
@@ -15,10 +15,12 @@ GC.get_data = function() {
 
     function processBoneAge(boneAgeValues, arr, units) {
         boneAgeValues && boneAgeValues.forEach(function(v){
-            arr.push({
-                date: v.effectiveDateTime,
-                boneAgeMos: units.any(v.valueQuantity)
-            })
+            if (v.status.toLowerCase() === 'final' || v.status.toLowerCase() === 'amended') {
```

Comment:
Would be nice if we can create a method : 
`
function validObservationObj(obj){
        if (obj.status && (obj.status.toLowerCase() === 'final' || obj.status.toLowerCase() === 'amended') &&
          obj.hasOwnProperty('valueQuantity') && obj.valueQuantity.hasOwnProperty('value') && obj.valueQuantity.hasOwnProperty('code')) {
          return true;
        }
       return false
      };
`

which will check for "status" and existence of valueQuantity before using it.